bundlers/webpack-browser-custom-output/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "webpack",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "jest": "27.5.1",
+    "prisma": "3.15.0-integration-12296-custom-path-pkg.1",
+    "webpack": "5.72.1",
+    "webpack-cli": "4.9.2"
+  },
+  "dependencies": {
+    "@prisma/client": "3.15.0-integration-12296-custom-path-pkg.1"
```

Comment:
I guess this test woudl be failing right now with the normal `dev` version of Prisma Client?

Reply:
Yes, I'll update it do dev version once prisma/prisma#13291 is fixed